alpha/scripts/utils/updatePartnerEntries2Sphinx.php

```diff
@@ -51,8 +51,16 @@
 		if ($mode == 'execute')
 		{
 			echo "Updating [". $item->getId() ."] ... "; 
-			$sphinx->saveToSphinx($item, false, true);
-			echo "Done\n";
+			try {
+				$sphinx->saveToSphinx($item, false, true);
+				echo 'Done' . PHP_EOL;
+			} catch (PDOException $e) {
```

Comment:
why PDOException specifically?
I think we would be better catch Exception and inside we can check, like
			catch (Exception $e)
			{
				if ($e instanceof PDOException && strpos($e->getMessage(), 'unknown column') !== false)
.
.